script.js

```diff
@@ -146,6 +149,7 @@ const stockMathInfo = async (year_number, month_number, day_number, symbol) => {
       }
 
 
+      // COMMENT(andymina)
```

Comment:
Might be worth making this whole loop into a separable function you can call. It would be easier to apply in different areas and would offer some good parameterization.